src/common/definitions.js

```diff
@@ -692,6 +851,15 @@ const uuid = {
   format: 'uuid',
 };
 
+const hcaEmail = {
+  type: 'string',
+  maxLength: 256,
```

Comment:
why is the maxLength 256?

Reply:
Sorry, I was just copying the `email` var in `defninitons`. I'll change it back:
```
const email = {
  type: 'string',
  maxLength: 256,
  format: 'email',
};
```